refresh.template.py

```diff
@@ -683,7 +694,36 @@ def _get_apple_active_clang():
     # Unless xcode-select has been invoked (like for a beta) we'd expect, e.g., '/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang' or '/Library/Developer/CommandLineTools/usr/bin/clang'.
 
 
-def _apple_platform_patch(compile_args: typing.List[str]):
+@functools.lru_cache(maxsize=None)
+def _get_apple_active_swiftc():
+    """Get path to xcode-select'd swiftc version."""
+    return subprocess.check_output(
+        ('xcrun', '--find', 'swiftc'),
+        stderr=subprocess.DEVNULL, # Suppress superfluous error messages like "Requested but did not find extension point with identifier..."
+        encoding=locale.getpreferredencoding()
+    ).rstrip()
+
+
+def _apple_swift_patch(compile_args: typing.List[str]):
+    """De-Bazel(rule_swift) the command into something sourcekit-lsp can parse."""
+
+    # Remove build_bazel_rules_swift/tools/worker/worker
+    compile_args.pop(0)
```

Comment:
Could you explain/doc a little bit more about what's happening here and below? I assume these are things built into the wrapper scripts, maybe with a persistent worker? (Sorry for my ignorance--I just haven't worked much with Bazel&swift)

Reply:
https://github.com/hedronvision/bazel-compile-commands-extractor/commit/c7a4df6b6854f5d9bbaeb163b04eeac8b5695454

Reply:
Got it. Is there a case we need to worry about where the user disables persistent workers?

Reply:
At least in the current rules_apple version we don't need. 
The [worker](https://github.com/bazelbuild/rules_swift/blob/master/tools/worker/worker_main.cc) has wrapped both.